geometry_msgs/Pose pose
float64[] joints
uint32[] conf_RLF
---
# Success if empty
string error
